src/pretix/plugins/sendmail/signals.py

```diff
@@ -1,9 +1,26 @@
+from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django.urls import resolve, reverse
 from django.utils.translation import gettext_lazy as _
 
+from pretix.base.models import SubEvent
 from pretix.base.signals import logentry_display
 from pretix.control.signals import nav_event
+from pretix.plugins.sendmail.models import ScheduledMail
+
+
+@receiver(post_save, sender=SubEvent)
```

Comment:
note to self: will need to check if this works properly with bulk-insertion of subevents, but probably will

Reply:
it does. just checked